src/webchat-ui/components/plugins/MessagePluginRenderer.tsx

```diff
@@ -39,6 +39,11 @@ export default ({ message, config, onSendMessage, plugins, isFullscreen, onSetFu
 
     const matchedPlugins = getPluginsForMessage(plugins)(message);
 
+    const regularMessageBot = 'regular-message regular-message-bot';
```

Comment:
the second class name can be just `bot`, not `...-bot`

Reply:
fixed